scissors/scissors_game/scissors.h

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: GPL-2.0
+static const char * const Game_Elements[] = {
+	"unknown", "rock", "paper", "scissors"};
+static enum {
+	UNKNOWN,
+	ROCK,
+	PAPER,
+	SCISSORS
+} element_id;
```

Comment:
Variable definitions (and thus - memory allocation) in header!

What's the purpose of `element_id` at all? - Most probably you've meant something different.